build.gradle

```diff
@@ -69,6 +72,10 @@ spotless {
         removeUnusedImports()
         importOrder()
     }
+    flexmark {
+        target "*.md"
+        flexmark()
+    }
```

Comment:
I'm not familiar with flexmark. Can you provide some context regarding why it should be added?

Reply:
It helps formatting the README